sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/AppendBlobClient.java

```diff
@@ -104,7 +143,13 @@ public final class AppendBlobClient extends BlobClientBase {
      */
     @Override
     public AppendBlobClient getEncryptionScopeClient(String encryptionScope) {
-        return new AppendBlobClient(appendBlobAsyncClient.getEncryptionScopeAsyncClient(encryptionScope));
+        EncryptionScope finalEncryptionScope = null;
+        if (encryptionScope != null) {
+            finalEncryptionScope = new EncryptionScope().setEncryptionScope(encryptionScope);
+        }
+        return new AppendBlobClient(this.appendBlobAsyncClient, getHttpPipeline(), getAccountUrl(), getServiceVersion(),
```

Comment:
This and `getCustomerProvidedKeyClient` need to update the asynchronous client as well.